spec/controllers/omniauth_callbacks_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe OmniauthCallbacksController, type: :controller do
```

Comment:
Ideally, spec files should have similar paths to the files they are testing, in this case:
`spec/controllers/users/omniauth_callbacks_controller_spec.rb`

```suggestion
RSpec.describe Users::OmniauthCallbacksController, type: :controller do
```